File: academic-system-python/academic_system/model/practical_assignment.py
from .assessment import Assessment


class PracticalAssignment(Assessment):
    # Cria uma avaliação do tipo Atividade Prática com valor e peso informados
    def __init__(self, value: float, weight: float):
        super().__init__(value, weight)

    # Retorna o identificador do tipo desta avaliação
    @property
    def type(self) -> str:
        return "Practical Assignment"

File: academic-system-java/src/main/java/org/example/academic/system/model/PracticalAssignment.java
package org.example.academic.system.model;

public class PracticalAssignment extends Assessment {
    // Cria uma avaliação do tipo Atividade Prática com valor e peso informados
    public PracticalAssignment(double value, double weight) {
        super(value, weight);
    }

    // Retorna o identificador do tipo desta avaliação
    @Override
    public String getType() {
        return "Practical Assignment";
    }
}
